### ISSUE TITLE
ErrorException when migrating database

### ISSUE THREAD

**Issue by damianlewis** [rainlab/blog-plugin#480]
I've just downloaded and installed version 1.3.4 of this plugin and when running `php artisan october:up` I get the following error. I've reverted back to version 1.3.2 which doesn't cause this error.

```bash
...
Cms
 - Migrating: 2014_10_01_000001_Db_Cms_Theme_Data
 - Migrated:  2014_10_01_000001_Db_Cms_Theme_Data
 - Migrating: 2016_10_01_000002_Db_Cms_Timestamp_Fix
 - Migrated:  2016_10_01_000002_Db_Cms_Timestamp_Fix
 - Migrating: 2017_10_01_000003_Db_Cms_Theme_Logs
 - Migrated:  2017_10_01_000003_Db_Cms_Theme_Logs
 - Migrating: 2018_11_01_000001_Db_Cms_Theme_Templates
 - Migrated:  2018_11_01_000001_Db_Cms_Theme_Templates
October.Demo
- v1.0.1:  First version of Demo
RainLab.Blog


  [ErrorException]
  Trying to get property 'id' of non-object
```

**Comment by bennothommo** [comment]
@damianlewis Would you be able to provide a stack-trace, or some indication as to where the exception is thrown? This will help us to locate the error.

**Comment by bennothommo** [comment]
Fixed by https://github.com/rainlab/blog-plugin/commit/aceda914bdbc1908af5e019d4e5fd09292bc0e37. Will be released with version 1.3.5 of Blog plugin.
